docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -452,6 +452,7 @@ control of your Kubernetes cluster.
 * Learn about kubeadm's advanced usage on the [advanced reference
   doc](/docs/admin/kubeadm/).
 * Learn more about Kubernetes [concepts](/docs/concepts/) and [`kubectl`](/docs/user-guide/kubectl-overview/).
+* Configure log rotation. You can use **logrotate** for that. In case of using docker you can specify log rotation options for docker demon, for example `--log-driver=json-file --log-opt=max-size=10m --log-opt=max-file=5`. See [Configure and troubleshoot the Docker daemon](https://docs.docker.com/engine/admin/) for details.
```

Comment:
suggested edit:

\* Configure log rotation. You can use \*\*logrotate\*\* for that. When using Docker, you can specify log rotation options for Docker daemon, for example \`--log-driver=json-file --log-opt=max-size=10m --log-opt=max-file=5\`. See \[Configure and troubleshoot the Docker daemon](https://docs.docker.com/engine/admin/) for more details.